specs/pvs/corla.pvs

```diff
@@ -863,6 +934,8 @@ BEGIN
 % @description The audited contest determines the number of ballots
 % that must be examined and verified during the RLA.
   number_of_ballots_to_audit: [audited_contest -> nat]
+
+% @todo kiniry Still need to model core RLA algorithm(s).
 END rlas
 
 %+END audits
```

Comment:
line 896/969 what real world verification of a cvr does this line model? I.e., what does it mean for a cvr to be verified or unverified?